app/components/draggable-object-target.js

```diff
@@ -3,8 +3,10 @@ import Droppable from 'ember-drag-drop/mixins/droppable';
 import log from '../helpers/log';
 
 export default Ember.Component.extend(Droppable, {
-  classNames: ["draggable-object-target"],
-  
+  classNameBindings: ['droppableClass'],
+
+  droppableClass: 'draggable-object-target',
```

Comment:
@SirZach @mharris717 one question on this line: Do you need to bind `droppableClass` since you just return a static string `draggable-object-target` for it? Other than that any chance of merging this?